$NetBSD: patch-ad,v 1.2.76.1 2011/12/07 08:30:51 tron Exp $

Fix PR 5746 by Benjamin Lorenz <lorenz@ps.uni-sb.de>.

--- misc.h.orig	1998-03-07 21:36:05.000000000 +0000
+++ misc.h
@@ -24,7 +24,7 @@ long random(void);
 
 #ifndef linux
 #if defined(BSD4_4) || defined(HPArchitecture) || defined(SGIArchitecture) || defined(_AIX)
-void srandom(unsigned int);
+/* void srandom(unsigned int); */
 #else
 int srandom(unsigned int);
 #endif	/* BSD4_4 */
@@ -97,7 +97,6 @@ void HelpInit(Widget top);
 void GetDefaultWH(int *w, int *h);
 char *GetDefaultRC(void);
 void SetIconImage(Widget w);
-void main(int argc, char *argv[]);
 
 /* misc.c */
 Widget GetToplevel(Widget w);
